code/game/objects/items/devices/swapper.dm

```diff
@@ -97,12 +103,12 @@
 
 	var/atom/movable/A = get_teleportable_container()
 	var/atom/movable/B = linked_swapper.get_teleportable_container()
-	var/turf/turf_A = get_turf(A)
-	var/turf/turf_B = get_turf(B)
+	var/target_A = drop_location(A)
+	var/target_B = drop_location(B)
```

Comment:
should be `X.drop_location()` not `drop_location(X)`

Reply:
whoops